PWGMM/Mult/Tasks/dndeta-mft.cxx

```diff
@@ -1,6 +1,6 @@
-// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
-// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
-// All rights not expressly granted are reserved.
+// Copyright 2020-2022 CERN and copyright holders of ALICE O2.
```

Comment:
Please also make sure the the copyright headers are not reformatted from their original state.